orca-redis/src/main/java/com/netflix/spinnaker/orca/pipeline/persistence/jedis/RedisExecutionRepository.java

```diff
@@ -503,6 +504,18 @@ Execution retrieveOrchestrationForCorrelationId(
     });
   }
 
+  @Override
+  public @Nonnull List<Execution> retrieveBufferedExecutions() {
+    // TODO rz - This is definitely not a healthy way to do this.
+    return Observable.concat(
```

Comment:
Lulz. Answers my earlier question re: how slow this is.

Reply:
We may want to think through what happens to an `ORCHESTRATION` that is initiated through the UI ... it's possible that _should not_ ever be buffered?

Reply:
Definitely, good thought! This would be pretty straight forward; create an `OriginBufferPolicy` that returns a `force=true` to `ENQUEUE` the execution.

Reply:
👍